### ISSUE TITLE
[Feature Request] Library persistent settings view / Save customized Library views

### ISSUE THREAD

**Issue by cesalo** [gotson/komga#190]
### Is your feature request related to a problem? Please describe.
Any ordering or details settings in the library view is not persistent. If I order a library by "Date Updated" for example and then go Home and back to the library it goes back to being ordered by "Name" which seems to be default.
### Describe the solution you'd like
If I pick "Date Updated" as the sorting option I'd like that is persistent and every time I go into this library is ordered by "Date Updated".
### Describe alternatives you've considered or other apps that can do what you want
Another related feature could be to add a way to save "customize" views .. for example let's say that I want to show just the "In Progress" Series or maybe "Ongoing" + "Unread" + "Order by Date Updated": I chose this option and "save" this view for a future use.

**Comment by gotson** [comment]
I don't want to persist those settings, because it would confuse most of the users (that are not power users).

All sort and filter settings are stored in the URL as query parameters, you can use a classic bookmark to restore those views.

**Comment by cesalo** [comment]
Gotcha .. I mentioned this following the Plex analogy. In Plex once you apply a filter or a sorting option it remains persistently (for your user) in any Library view.
For example I always have my TV-Shows ordered by "Date Added" so I can check quickly in case  I forgot to grant permissions to access new TV-Shows.

**Comment by gotson** [comment]
I never really paid attention. I could store it in cookies?

**Comment by cesalo** [comment]
Yes .. that could be a good option. Generally you use the same browser and if not you can set it up again but at least it would be persistent until you change browsers.

**Comment by gotson** [comment]
the page size is already saved like that. And it's not really an important setting, so it's not a big deal if it's lost because the cookies have been removed.

**Comment by gotson** [comment]
> Gotcha .. I mentioned this following the Plex analogy. In Plex once you apply a filter or a sorting option it remains persistently (for your user) in any Library view.
> For example I always have my TV-Shows ordered by "Date Added" so I can check quickly in case I forgot to grant permissions to access new TV-Shows.

I just tested Plex, the filters/sort are kept per library, not across libraries.

**Comment by cesalo** [comment]
Correct .. Mixing this between libraries could be confusing. I just have 1 at the moment.
